core/src/test/java/org/elasticsearch/search/aggregations/AggregationsBinaryIT.java

```diff
@@ -41,6 +42,8 @@
 import static org.hamcrest.core.IsNull.notNullValue;
 
 @ESIntegTestCase.SuiteScopeTestCase
+@AwaitsFix(bugUrl = "needs fixing after the search request refactor. Do we need agg binary?")
```

Comment:
I guess we need agg_binary as long as we need query_binary etc. ? not sure how many users use that stuff though.